app/scripts/src/content/search.js

```diff
@@ -59,20 +59,24 @@ Search.prototype = {
   findEpisode: function(options) {
     this.findItem({
       success: function(response) {
-        Request.send({
-          method: 'GET',
-          url: this.getEpisodeUrl(response['show']['ids']['slug']),
-          success: function(resp) {
-            if (this.item.episode) {
-              options.success.call(this, JSON.parse(resp));
-            } else {
-              this.findEpisodeByTitle(resp, options);
+        if (!response) {
```

Comment:
But if the response is `undefined` here, I don't see why send it to the callback. Maybe what you did on [sendAnalyticsEvent](https://github.com/tegon/traktflix/pull/18/files?diff=split#diff-8043154e14de355161517a54b7c405fdR34) is enough, you can use `item.title` to log to the console also.